sechub-shared-kernel/src/main/java/com/mercedesbenz/sechub/sharedkernel/messaging/MessageID.java

```diff
@@ -258,6 +258,12 @@ public enum MessageID {
      */
     ASSIGN_OWNER_AS_USER_TO_PROJECT(MessageDataKeys.PROJECT_TO_USER_DATA),
 
+    /**
+     * Request all execution profile IDs for each project. This is sent when a user
+     * requests information about all projects the user is part of.
+     */
+    REQUEST_PROFILE_IDS_FOR_PROJECT(MessageDataKeys.PROJECT_ASSIGNED_PROFILE_IDS, MessageDataKeys.PROJECT_IDS),
+
     ;
```

Comment:
As mentioned before we need an answer mesage id here 
RESULT_PROFILE_IDS_FOR_PROJECT(...)